server/api/events.js

```diff
@@ -1,10 +1,10 @@
 const router = require('express').Router()
-const {Event} = require('../db/models')
+const {Event, User} = require('../db/models')
 module.exports = router
 
-router.get('/', async (req, res, next) => {
+router.get('/all', async (req, res, next) => {
```

Comment:
I don't understand this.  Usually in REST a / route without anything more implies 'all'